Modules/timemodule.c

```diff
@@ -2390,3 +2389,67 @@ pysleep(PyTime_t timeout)
     return -1;
 #endif
 }
+
+
+// time.sleep(0) optimized implementation.
+// On error, raise an exception and return -1.
+// On success, return 0.
+//
+// Rationale
+// ---------
+// time.sleep(0) accumulates delays in the generic implementation, but we can
```

Comment:
"accumulates delays" I don't understand this part.

Reply:
Ah right, `time.sleep(0)` itself is not accumulating delays. I'll rewrite that part tomorrow or during the week.